EIPS/eip-5573.md

```diff
@@ -1,6 +1,6 @@
 ---
 eip: 5573
-title: Sign-In with Ethereum Capabilities (CapGroks)
+title: Sign-In with Ethereum Capabilities, CapGroks
```

Comment:
```suggestion
title: Sign-In with Ethereum Capabilities
```

Reply:
If you're okay with it @Pandapip1, I'm fine leaving CapGroks in the title. It's more searchable.

Reply:
What punctuation should be used then? I would say that `CapGroks: Sign-In with Ethereum Capabilities` would be ideal but that won't work because YAML.